public/alternative-routes/index.html

```diff
@@ -0,0 +1,82 @@
+<!DOCTYPE html>
+<html lang="nl">
+  <head>
+    <title>Chargetrip API: route example</title>
+    <meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
+    <meta name="description" content="Build a route with the Chargetrip API" />
+    <meta name="keywords" content="Chargetrip, Chargetrip API, EV, ev-cars, ev-routing" />
+    <meta name="viewport" content="initial-scale=1,maximum-scale=1,user-scalable=no" />
+    <link href="https://fonts.googleapis.com/css2?family=Inter:wght@100;200;400;600&display=swap" rel="stylesheet" />
+    <link href="https://api.mapbox.com/mapbox-gl-js/v1.10.0/mapbox-gl.css" rel="stylesheet" />
+    <link rel="icon" href="../favicon-57.png" type="image/x-icon" />
+    <link rel="stylesheet" href="style.css" />
+  </head>
+
+  <body>
+    <div id="map"></div>
+    <div class="github-link">
+      <a href="https://github.com/chargetrip/examples/tree/master/alternative-routes" title="github project">
+        View on Github
+      </a>
+      <button aria-label="remove github link" title="remove github link" onclick="this.parentElement.remove()"></button>
+    </div>
+
+    <div class="journey-specs">
+      <header>
+        <span class="duration" id="duration"></span>
+        <h1>Amsterdam • Neubrandenburg</h1>
+        <span id="loader">Loading...</span>
+        <div class="tags">
+          <div id="distance" class="tag"></div>
+          <div id="stop" class="tag"></div>
+          <div id="consumption-overview" class="tag"></div>
+        </div>
+      </header>
+
+      <section>
+        <div class="route-selector" id="selector">
+          <input class="checkbox-routes" type="radio" name="routes" id="route-0" checked />
```

Comment:
Very nice that you grouped them together via 'name' property! Then in JS it gives you an advantage ;)